qa/rolling-upgrade/src/test/java/org/elasticsearch/upgrades/FeatureUpgradeIT.java

```diff
@@ -92,8 +93,8 @@ public void testGetFeatureUpgradeStatus() throws Exception {
                     .findFirst()
                     .orElse(Collections.emptyMap());
 
-                assertThat(feature.size(), equalTo(4));
-                assertThat(feature.get("minimum_index_version"), equalTo(UPGRADE_FROM_VERSION.toString()));
+                assertThat(feature, aMapWithSize(4));
+                assertThat(feature.get("minimum_index_version"), equalTo(Integer.toString(UPGRADE_FROM_VERSION.id)));
```

Comment:
The upgrade version is specified in a env parameter; we'll need to add index version to the test infrastructure as well I think...

Reply:
#97200